app/app.go

```diff
@@ -426,6 +427,13 @@ func New(
 		app.BankKeeper,
 	)
 
+	app.oracleKeeper = *oraclekeeper.NewKeeper(
```

Comment:
I just wonder do you have any specific reason to move oracle keeper before wasm??

Reply:
No. I simply wanted to put our modules together in one place. :)